lib/Convos/Core/Connection/Irc.pm

```diff
@@ -435,8 +432,7 @@ sub _steal_nick {
   my $self = shift;
   my $tid;
 
-  Scalar::Util::weaken($self);
-  $tid = $self->_irc->ioloop->recurring(
```

Comment:
I don't get this either: If I avoid using the `ioloop` from Mojo::IRC, then I also avoid a memory cycle..?

Also, why is `weaken()` not required above?